api/v1alpha3/condition_consts.go

```diff
@@ -26,6 +26,11 @@ const (
 
 // Common ConditionReason used by Cluster API objects.
 const (
+	// WaitingForControlPlaneAvailableReason (Severity=Info) document an object waiting for the control plane to be available.
+	// e.g. Having the control plane machine available is a pre-condition for joining additional control planes
+	// or workers nodes.
+	WaitingForControlPlaneAvailableReason = "WaitingForControlPlaneAvailable"
```

Comment:
This const exists in many providers, so I'm moving IT in the main CAPI API and deprecate local ones (in CAPD, CAPA, CAPV)